### DISCUSSION TITLE
Users creation on the platform

### DISCUSSION THREAD

**Discussion by martinambrueso** [apache/devlake#8228] [General]
Hi guys, I hope you are well. I have a quick question: how can I create users on the platform? Is it possible to integrate this system with LDAP to manage users and roles? Does any release allow this? Thanks!

**Comment by klesh** [comment]
There isn't a built-in user management system. If you require one, you might want to explore [oauth2_proxy](https://github.com/oauth2-proxy/oauth2-proxy).
